cmake/configure/configure_2_trilinos.cmake

```diff
@@ -188,12 +188,12 @@ ENDMACRO()
 
 
 MACRO(FEATURE_TRILINOS_CONFIGURE_EXTERNAL)
-  SET(DEAL_II_EXPAND_TRILINOS_VECTOR "TrilinosWrappers::Vector")
-  SET(DEAL_II_EXPAND_TRILINOS_BLOCKVECTOR "TrilinosWrappers::BlockVector")
+  SET(DEAL_II_EXPAND_TRILINOS_VECTOR "TrilinosWrappers::VectorBase")
```

Comment:
Should we expand `VectorBase` here? We do not do this for the PETSc wrappers, but I am not familiar enough with Trilinos to know for sure in this case.